src/main/java/org/jitsi/impl/protocol/xmpp/ChatRoom.java

```diff
@@ -198,4 +199,40 @@ void join()
      * Get the unique meeting ID associated by this room (set by the MUC service).
      */
     String getMeetingId();
+
+    /**
+     * Whether A/V moderation is enabled.
+     * @param mediaType the media type.
+     * @return whether A/V moderation is enabled.
+     */
+    boolean isAvModerationEnabled(MediaType mediaType);
+
+    /**
+     * Sets new value for A/V moderation.
+     * @param actorJid the jid of the participant performing the operation.
+     * @param mediaType the media type.
+     * @param value the new value.
+     */
+    void setAvModerationEnabled(Jid actorJid, MediaType mediaType, boolean value);
+
+    /**
+     * Returns the actor that performed the last enable/disable action.
+     * @param mediaType the media type.
+     * @return the jid of the actor.
+     */
+    Jid getAvModerationActor(MediaType mediaType);
```

Comment:
This is the participant we had stored, that had enabled the AV moderation for that mediaType.

Reply:
And yes, this is to be passed to handleMuteRequest only. As it can happen when that is enabled, but we store it for all participants which join later and we need to use it later.

Reply:
Hum, good point ... which I'm not sure how to handle that ...